resources/definitions/syndaveraxi.def.json

```diff
@@ -0,0 +1,46 @@
+         },
+        "machine_end_gcode": {
+            "default_value": "M400                                               ; wait for moves to finish\nM140 S{material_part_removal_temperature}          ; start bed cooling\nM104 S0                                            ; disable hotend\nM107                                               ; disable fans\nM117 Cooling please wait\nG91                                                ; relative positioning\nG1 Z5 F3000                                        ; move Z up 5mm so it wont drag on the print\nG90                                                ; absolute positioning\nG1 X5 Y5 F3000                                     ; move to cooling position\nM190 R{material_part_removal_temperature}          ; wait for bed to cool down to removal temp\nG1 X145 Y260 F1000                                 ; present finished print\nM140 S{material_keep_part_removal_temperature_t}   ; keep temperature or cool down\nM77                                                ; End LCD Print Timer\nM18 X Y E                                          ; turn off x y and e axis\nM117 Print Complete."
```

Comment:
The settings "material_part_removal_temperature" and "material_keep_part_removal_temperature_t" mentioned in the end gcode don't exist. 
Cura complains: 
```
Unable to replace 'material_part_removal_temperature' placeholder in start/end g-code
Unable to replace 'material_keep_part_removal_temperature_t' placeholder in start/end g-code
```